lib/rmt/cli/export.rb

```diff
@@ -10,7 +10,8 @@ def data(path)
   desc 'settings PATH', 'Store repository settings at given path'
   def settings(path)
     needs_path(path) do
-      File.write(File.join(path, 'repos.json'), Repository.only_mirrored.pluck(:id).to_json)
+      data = Repository.only_mirrored.inject([]) { |data, repo| data << { url: repo.external_url, auth_token: repo.auth_token.to_s } }
```

Comment:
That doesn't seem to do what it needs to.

For SCC repos dumping just the ids was fine, because all the data about those repos is contained in the `scc-data` JSON dump.

For custom repos, however, there's no info about the repos in the JSON dump -- and the current code wouldn't preserve ids, names and associations with products.

Reply:
We've established that this works correctly :+1: